src/cmd/library_publish.js

```diff
@@ -66,32 +53,30 @@ export class LibraryPublishCommand extends Command {
 	 * @returns {Promise} To run the library publish command.
 	 */
 	run(state, site) {
-		const events = (event, ...args) => {
-			const fn = site[event].bind(site) || (() => {});
-			fn(...args);
-		};
+		let name;
+		return Promise.resolve(site.libraryIdent())
+			.then(ident => name = ident)
```

Comment:
Yeah, I like the `then` under the `return` so you see a logical sequence, but WebStorm insists on adding the indent. There might be a setting to fix this.